btcjson/chainsvrresults.go

```diff
@@ -379,6 +382,48 @@ type GetRawMempoolVerboseResult struct {
 	Depends          []string `json:"depends"`
 }
 
+// ZmqNotificationResult models the data returned from the zmqnotifications command.
+type ZmqNotificationResult struct {
```

Comment:
Better stick to the standard naming convention, and use `GetZmqNotificationsResult`.